__tests__/app.test.js

```diff
@@ -41,4 +42,26 @@ describe('app testing', () => {
             })
         })
     })
+    describe('get article by id', () => {
+        test('Returns the requested article', () => {
```

Comment:
This test is not actually testing what the description (correctly 😄 ) says it should be testing. We need to make our assertions check the article is indeed the one requested. ObjectContaining is the right matcher to use but within there we need to be more specific for each value of the properties.